src/Model/GContact.php

```diff
@@ -1343,14 +1343,18 @@ public static function discoverFollowers(string $url)
 			foreach ($contacts as $contact) {
 				$gcontact = DBA::selectFirst('gcontact', ['id'], ['nurl' => Strings::normaliseLink(($contact))]);
 				if (DBA::isResult($gcontact)) {
+					$field = [];
```

Comment:
```suggestion
					$fields = [];
```

Reply:
Möööp

Reply:
That's a new one. I knew "Örgs" but not "Möp".

Reply:
I'm a bag of surprise for new words :-)